resources/views/application/edit.blade.php

```diff
@@ -37,6 +37,19 @@
             </a>
         </div>
 
+        @if ($invitingApplication)
+            <div class="alert alert-info text-center">
+                <div class="w-50 mx-auto">
+                    <h3>Confirm Invitation</h3>
+                    <p>
+                        To finalize accepting your invitation from <em>{{ $invitingApplication->getFullName() }}</em>
```

Comment:
Same, suggestion:

To accept your invitation from ... (...) and become part of their dealership as Share, please review the data below and click on "Submit your application".